src/cs/usfca/edu/histfavcheckout/controller/HistFavCheckoutHandler.java

```diff
@@ -254,6 +256,50 @@ public OperationalResponse checkout(OperationalRequest request) {
 		return resp;
 	}
 
+	public ResponseEntity<?> totalFavesAndCheckouts(int userId) {
```

Comment:
Can we include pagination support? Sorry we forgot to include this in the original spec. It's just a matter of including page & nums in the method parameters, and modifying the query where you retrieve all movies favorited by the user to support pagination (more on this in my second comment below)

You can look at the getCheckouts() method, you just need to mirror these lines : [97](https://github.com/usfcs-perf-eng-s20/heroku-project-favs/blob/master/src/cs/usfca/edu/histfavcheckout/controller/HistFavCheckoutHandler.java#L97), [99-100](https://github.com/usfcs-perf-eng-s20/heroku-project-favs/blob/master/src/cs/usfca/edu/histfavcheckout/controller/HistFavCheckoutHandler.java#L99-L100)